tests/test_hbavss.py

```diff
@@ -43,7 +43,38 @@ def get_avss_params(n, t):
 
 
 @mark.asyncio
-async def test_hbavss_client_mode(test_router):
+async def test_hbavss_batch(test_router):
```

Comment:
Can you create a test case which does multiple batch AVSSes from one node in parallel? Only one dealer node, which does two avsses in parallel. This will ensure that there are no tag conflicts or any other id conflicts and that messages are getting routed to the correct AVSS instance.

Reply:
Will do it later.

Reply:
Sure!